pkgs/tools/misc/spigot/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, buildPackages
+, fetchgit
+, autoreconfHook
+, gmp
+, ncurses
+, halibut
+, perl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "spigot";
+  version = "20200101";
+  src = fetchgit {
+    url = "https://git.tartarus.org/simon/spigot.git";
+    rev = "b1b0b20";
```

Comment:
Oh, I had assumed that upstream had git tags for their releases. Most projects have that (e.g. a tag `v20200101` pointing to the right rev). Without tags, the original argument for moving to `fetchgit` isn't really valid anymore. We need to look up the right commit hash when updating, so automatic updates won't work. That's unfortunate.

You can stay with `fetchgit` or revert to `fetchurl`, whatever you prefer. If you stay with `fetchgit`, please use a fully-qualified rev though (not a shortened one). If you want to go the extra mile, contact the upstream maintainer and ask them nicely if they would consider adding tags for releases.

Reply:
Or alternatively ask them to provide a download link that only includes the version, not the hash.

Reply:
Okay, full rev is in there.  I'll reach out to Simon and see what he says about tagging.  In the mean time, anything else?